src/fosslight_source/run_scancode.py

```diff
@@ -47,14 +48,36 @@ def main():
                 output_file = arg
             elif opt == "-m":
                 print_matched_text = True
+            elif opt == "-x":
+                run_scanoss = True
     except Exception:
         print_help_msg_source()
 
     timer = TimerThread()
     timer.setDaemon(True)
     timer.start()
-    run_scan(path_to_scan, output_file, write_json_file, -1, False, print_matched_text)
+    if run_scanoss == True:
+        run_scanoss_py(path_to_scan, output_file)
+    else:
+        run_scan(path_to_scan, output_file, write_json_file, -1, False, print_matched_text)
+
+def run_scanoss_py(path_to_scan, output_file_name=""):
+    scan_command = "scanoss-py scan -o "
+
+    _windows = platform.system() == "Windows"
```

Comment:
Why are you checking the window?

Reply:
When analyzing with Scancode, there is no input path, and in case of Windows, the current path is analyzed. But this is legacy that should be removed.